migrator/clone/rocksdb/db_clone_manager_impl.go

```diff
@@ -327,27 +334,61 @@ func (d *dbCloneManagerImpl) GetDirName(cloneName string) string {
 func (d *dbCloneManagerImpl) DecommissionRocksDB() {
 	log.Info("DecommissionRocksDB")
 	for k := range d.cloneMap {
-		path := d.getPath(k)
-		if exists, err := fileutils.Exists(path); err != nil {
+		dirPath := d.getPath(k)
+		if exists, err := fileutils.Exists(dirPath); err != nil {
 			log.Error(err)
 			continue
 		} else if exists {
-			log.Infof("Removing database %s", path)
-			linkTo, err := fileutils.ResolveIfSymlink(path)
-			if err != nil {
-				log.Error(err)
-				continue
-			}
-			log.Infof("Remove path = %q", path)
-			if err = os.RemoveAll(path); err != nil {
-				log.Error(err)
-				continue
-			}
-			// Remove any rocks database if in postgres mode
-			log.Infof("Remove linkTo = %q", linkTo)
-			if err = os.RemoveAll(linkTo); err != nil {
-				log.Error(err)
-				continue
+			// TODO(ROX-9882): While there is still stuff on the PVC there is an expectation that current
+			// exists in a certain state.  So until we can remove the rest of the components on the PVC, it
+			// is safer to put the contents in an obviously deprecated location.
+			if k == CurrentClone {
+				// Get the location Current points to
+				linkTo, err := fileutils.ResolveIfSymlink(dirPath)
+				if err != nil {
+					log.Error(err)
+					continue
+				}
+				if linkTo == d.getPath(decommissionedCurrent) {
+					log.Info("we have already moved to the RocksDB current to be decomissioned")
+					continue
+				}
+				// Remove the migrations versions to ensure we do not try to use this data for migrations
+				migrationVersionPath := filepath.Join(dirPath, migrations.MigrationVersionFile)
+				if err = os.RemoveAll(migrationVersionPath); err != nil {
```

Comment:
You can remove the entire directory here. migrationVersionPath is only the yaml file.